lua/lspconfig/server_configurations/nim_langserver.lua

```diff
@@ -0,0 +1,23 @@
+local util = require 'lspconfig.util'
+
```

Comment:
add windows support here .

Reply:
Thank you for the review. Could you give me a pointer about what is required for windows support. I based this on nimls.lua, so I guess the same change would be needed in that module for windows support as well?